### DISCUSSION TITLE
Preview of email messages

### DISCUSSION THREAD

**Discussion by ShilpaSharePoint** [microsoft-search/pnp-modern-search#2475] [Q&A]
Hi, as a part of search we are displaying email messages subject, sent by and date sent by using Graph Search API. Part of our requirement we want to display preview of email message when we place cursor on the search results-message. But IFrame does not support to display email as preview. so wanted to check similar way like document preview we can get any other property or any other control which supports email message preview. Any help on this will be helpful,

**Comment by wobba** [comment]
You have to ask this on Stack Overflow for the Graph API, or on some Exchange/Outlook forum to figure out what URL you could use.
